src/HotChocolate/Core/src/Types.Scalars/IPv4Type.cs

```diff
@@ -0,0 +1,114 @@
+using System.Text.RegularExpressions;
+using HotChocolate.Language;
+
+namespace HotChocolate.Types.Scalars
+{
+    /// <summary>
+    /// The `IPv4` scalar type represents a valid a IPv4 address as defined
+    /// here https://en.wikipedia.org/wiki/IPv4.
+    /// </summary>
+    public class IPv4Type : StringType
+    {
+        private static readonly string _validationPattern =
+            ScalarResources.IPv4Type_ValidationPattern;
```

Comment:
I think this should rather be a constant since it is not translatable.

Reply:
Good point! 
Should I apply this to the other scalar types? 
I copied the pattern from other Regex-based scalar types, so for consistency, we should apply this to them too, IMHO.